packages/compile-vyper/index.js

```diff
@@ -130,6 +130,13 @@ const Compile = {
     });
   },
 
+  async sourcesWithDependencies({ paths = [], options }) {
+    return await Compile.sources({
+      sources: paths,
```

Comment:
This exposes the fact that compile-vyper's `sources` method doesn't conform to the type defined by compile-common